app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -182,11 +182,11 @@ You have the right to apply to train to teach in England if you are resident in
 * [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
 * [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
-### Deadlines for applications
+### Deadline for applications
 
-Applications to train to teach in 2022/23 have now closed. Applications for 2023/24 open on 11 October 2022.
+You can apply for teacher training courses starting in autumn 2023 until [September 2023](https://www.apply-for-teacher-training.service.gov.uk/provider/service-guidance/dates-and-deadlines).
```

Comment:
```suggestion
You can [apply for teacher training courses](https://www.gov.uk/apply-for-teacher-training) starting in autumn 2023 until September 2023.
```
For accessibility reasons, we should have more descriptive link text than 'September 2023', and looking at the page you've linked to, I'm not sure it's necessarily extra info users would need? Is it maybe enough just to give them the date applications close? (Especially because the link goes to Manage, which is a provider facing service rather than for candidates.)